test/h3core.spec.js

```diff
@@ -15,7 +15,7 @@
  */
 
 import test from 'tape';
-import * as h3core from '../lib/h3core.js';
+import * as h3 from '../lib/h3core.js';
```

Comment:
This change introduced a lot of noise into this file - the only new code here are the tests for the new functions.